en/micro-integrator/docs/setup/security/securing_management_api.md

```diff
@@ -89,20 +89,22 @@ enable = false
 
 ## Authorization
 
-Authorization can be set to resources that only need to be invoked by admin users. The resource `/management/users` is by default secured with authorization, meaning that only users with admin privileges can access this resource.
+You can enable Authorization only for resources (in the management API) that need to be invoked by admin users. The `/management/users` resource is by default secured with authorization, meaning that only users with admin privileges can access this resource.
```

Comment:
I don't think we need to add the part (in the management API) since this is the management api doc. Besides, they can enable the handler to others internal apis aswell.

Reply:
resolved